scrips/generate_pkgbuild.py

```diff
@@ -25,22 +25,11 @@ class Pkgbuild:
 
     dependencies = {
         "zivid-telicam-sdk": (),
-        "zivid": ("zivid-telicam-sdk",),
+        "zivid": ("zivid-telicam-sdk", "opencl-driver"),
```

Comment:
What's this `opencl-driver`?

Reply:
Meta package, opencl-nvidia for example fulfills this dependency.